workflows/templates/qdm.yaml

```diff
@@ -131,119 +137,6 @@ spec:
                   value: "{{ inputs.parameters.last-year }}"
 
 
-    - name: preprocess
-      inputs:
-        parameters:
-          - name: in-zarr
-          - name: out-zarr
-            value: "gs://scratch-170cd6ec/{{ workflow.uid }}/{{ pod.name }}/preprocessed.zarr"
-          - name: regrid-method
-          - name: domain-file
-          - name: correct-wetday-frequency
-          - name: apply-dtr-minimum-threshold
-          - name: add-cyclic
-            value: "lon"
-      outputs:
-        parameters:
-          - name: out-zarr
-            value: "{{ inputs.parameters.out-zarr }}"
-      script:
-        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.12.0
-        command: [ python ]
-        source: |
-          import logging
-          import dodola.services
-
-          logging.basicConfig(level=logging.INFO)
-          logger = logging.getLogger(__name__)
-
-          input_zarr = "{{ inputs.parameters.in-zarr }}"
-          apply_minimum_threshold = "{{ inputs.parameters.apply-dtr-minimum-threshold }}".lower() == "true"
-          correct_wetday_frequency = "{{ inputs.parameters.correct-wetday-frequency }}".lower() == "true"
-          if correct_wetday_frequency:
-              logger.info("Applying wet-day frequency correction")
-              wdf_corrected_zarr = "/workspace/wdf_corrected.zarr"
-              dodola.services.correct_wet_day_frequency(
-                  input_zarr,
-                  process="pre",
-                  out=wdf_corrected_zarr
```

Comment:
@brews, quoting your description about the PR : "This is an update from a single step writing intermediate output to the boot disk". This boot disk writing you're referring to is what happens in this line writing to `/workspace/...`?

You figured this was causing the instability ?

Reply:
@emileten Good question. Yes, each pod currently has about 100 GB attached to it, so we were using about 20 GBs of that to pass data around.

I do not think it was writing to local disk, specifically, that was the cause for the problems. Previously (like the tasmax runs) we were able to complete work with far fewer of these sporadic timeouts. It has only become a problem with DTR and its minimum threshold. So I don't have enough evidence to point to one specific thing as the cause for the bug. But, I also don't see evidence that writing to disk was to blame.